tests/test_en.py

```diff
@@ -131,6 +131,194 @@ def test_to_currency(self):
             "four pesos and one cent"
         )
 
+        self.assertEqual(
```

Comment:
the name of the file is `test_en.py` it is for English.

What about using a `*es.py` ?

Reply:
You right, it's fixed.